gettingStarted.md

```diff
@@ -51,6 +51,9 @@ This broker implements support for the
 [Asynchronous Service Operations](https://docs.cloudfoundry.org/services/api.html#asynchronous-operations),
 and calls AWS APIs to provision EC2 VMs.
 
+[Storage Service Operations](https://github.com/leonwanghui/opensds-broker/tree/master),
```

Comment:
Shouldn't this just be the URL to the repo? (Meaning: without `/tree/master`)

Reply:
i need to edit the link for the official repo address in opensds community repo, but it will also contain tree/master in order to visit the page